app/components/find/courses/entry_requirements_component/view.html.erb

```diff
@@ -3,32 +3,41 @@
   <h3 class="govuk-heading-m">Qualifications needed</h3>
   <div data-qa="course__required_qualifications">
 
-    <% if course.degree_grade.nil? && course.additional_degree_subject_requirements.nil? %>
+    <% unless course.teacher_degree_apprenticeship? %>
+      <% if course.degree_grade.nil? && course.additional_degree_subject_requirements.nil? %>
         <%= render CoursePreview::MissingInformationComponent.new(course:, information_type: :degree, is_preview: preview?(params)) %>
-    <% else %>
-      <p class="govuk-body"><%= degree_grade_content(course) %></p>
+      <% else %>
+        <p class="govuk-body"><%= degree_grade_content(course) %></p>
 
         <% if course.degree_subject_requirements.present? %>
           <p class="govuk-body">
             <%= helpers.markdown(course.degree_subject_requirements) %>
           </p>
-      <% end %>
+        <% end %>
 
-      <% if course.secondary_course? %>
-        <% if course.engineers_teach_physics? %>
+        <% if course.secondary_course? %>
+          <% if course.engineers_teach_physics? %>
+            <p class="govuk-body">
+              This <%= govuk_link_to "Engineers teach physics", t("find.get_into_teaching.url_engineers_teach_physics") %> course is designed for candidates who have a background in materials science and engineering. If your degree is in physics, please apply to our physics course.
+            </p>
+          <% end %>
+        <% end %>
+
+        <% if subject_knowledge_enhancement_content? %>
           <p class="govuk-body">
-            This <%= govuk_link_to "Engineers teach physics", t("find.get_into_teaching.url_engineers_teach_physics") %> course is designed for candidates who have a background in materials science and engineering. If your degree is in physics, please apply to our physics course.
+            If you need to improve your subject knowledge, you may be asked to complete a <%= govuk_link_to "subject knowledge enhancement (SKE) course.", t("find.get_into_teaching.url_subject_knowledge_enhancement") %>
           </p>
         <% end %>
       <% end %>
+    <% end %>
 
-      <% if subject_knowledge_enhancement_content? %>
-        <p class="govuk-body">
-          If you need to improve your subject knowledge, you may be asked to complete a <%= govuk_link_to "subject knowledge enhancement (SKE) course.", t("find.get_into_teaching.url_subject_knowledge_enhancement") %>
-        </p>
-      <% end %>
-     <% end %>
-    </p>
+    <% if course.teacher_degree_apprenticeship? %>
+      <p class="govuk-body">A levels</p>
+
+      <%= render Find::Courses::ALevelComponent::View.new(course:) %>
+
+      <p class="govuk-body">GCSEs</p>
+    <% end %>
```

Comment:
Should A levels and GCSEs be headings? (h4) rather than paragraphs?
<img width="751" alt="image" src="https://github.com/DFE-Digital/publish-teacher-training/assets/44073106/f30ff99d-c91e-4aa4-a144-40107328e525">

Reply:
The card and the Lucid was showing a paragraph. I will ask the designer and content.

Reply:
I don't think we actually provided a specification for what the information would look like in Find apologies. 

It would be an interim solution, as we have a future dev card for all course pages (TDA and non TDA) to convert this section into a description list:

| Example of non TDA course with new description list design |
| --- |
| ![Screenshot 2024-07-02 at 13 47 44](https://github.com/DFE-Digital/publish-teacher-training/assets/6122118/e9a73e54-b154-4f06-b271-a0f4a6584f1c) |

But for this PR I agree with @elceebee I think converting 'A level's and 'GCSEs' to (h4) would be good to do.

Reply:
Done!